apps/web/src/studio/hooks/useBridgeConnectionStatus.ts

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react';
+import { ConnectionStatus } from '../types';
+import { useHealthCheck } from './useBridgeAPI';
+
+export const useBridgeConnectionStatus = () => {
+  const { data, isFetching, error, bridgeURL } = useHealthCheck();
+
+  const status = useMemo<ConnectionStatus>(() => {
+    if (isFetching) {
+      return 'loading';
```

Comment:
Please use the connection status enum.

Reply:
@SokratisVidros Where is this enum? This is taken from the hook you wrote

Reply:
@SokratisVidros That is being used, but it's not an enum

Reply:
@SokratisVidros  That is being used, but it's not an enum!

Reply:
It's also typed based on the useMemo